src/mint_json.cr

```diff
@@ -14,34 +14,42 @@ module Mint
       end
     end
 
-    @dependencies = [] of Mint::Installer::Dependency
-    @formatter_config = Formatter::Config.new
     @parser = JSON::PullParser.new("{}")
-    @source_directories = [] of String
-    @test_directories = [] of String
-    @external_files = {
-      "javascripts" => [] of String,
-      "stylesheets" => [] of String,
+
+    getter dependencies = [] of Mint::Installer::Dependency
+    getter formatter_config = Formatter::Config.new
+    getter source_directories = %w[]
+    getter test_directories = %w[]
+    getter external_files = {
+      "javascripts" => %w[],
+      "stylesheets" => %w[],
     }
-    @application = Application.new
-    @name = ""
+    getter application = Application.new
+    getter name = ""
+    getter root : String
 
     json_error MintJsonRootNotAnObject
     json_error MintJsonRootInvalidKey
 
-    getter test_directories, source_directories, dependencies, application
-    getter external_files, name, root, formatter_config
-
     def self.parse_current : MintJson
       path = File.join(Dir.current, "mint.json")
       new File.read(path), Dir.current, path
-    rescue exception : IO::Error
+    rescue ex : IO::Error
```

Comment:
What is wrong with `exception` ? :smile:

Reply:
Oops 😅